### PR TITLE
MAR-4055-frontend-change-brief-to-opportunity

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

**Comment by Reshma-M-A** [comment]
There are some instances of briefs found under
`BriefDownloadResponses.js`
`BriefSubmitted.js`

**Review by Reshma-M-A — COMMENTED** [review]
Found instances of brief in:
apps/marketplace/components/Reports/ReportItem.js
apps/marketplace/components/Opportunity/OpportunitySpecialistInfoCard.test.js
apps/marketplace/components/BuyerATM/BuyerATMAboutStage.js
apps/marketplace/components/Brief/Overview/BriefOverview.js

**Review by Reshma-M-A — COMMENTED** [review]
Check BriefDownloadDocuments.js (line 12), BriefDownloadResponses.test.js, BuyerDashboardBriefTable.js (line 102), BuyerDashboardHelp.js (line 12) and BuyerOnboarding.js

**Comment by Reshma-M-A** [comment]
This branch has some conflicts.  @Revit can you please fix this conflict. Let me know if you have any issues.

File: apps/marketplace/components/Reports/ReportItem.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ const ReportItem = props => {
...
     const imgClass = includes(
       [
-        '768 Briefs by phase',
-        '767 Briefs by phase',
-        '768 Average responses per brief',
-        '767 Average responses per brief',
+        '768 Opportunities by phase',
+        '767 Opportunities by phase',
+        '768 Average responses per opportunity',
+        '767 Average responses per oppportunity',
```
[the remark this answers is not in the document]
Comment (Revit) [reply apps/marketplace/components/Reports/ReportItem.js]: changed as suggested

File: apps/marketplace/pages/ReportPage.js [hunk 2]
```diff
@@ -36,15 +36,15 @@ const itemData = {
       subitems: [
         {
           text: null,
-          image: 'Percentage of briefs open to all.svg',
-          mobileImage: 'Percentage of briefs open to all.svg',
-          imageAlt: '68% of all briefs have been open to all'
+          image: 'Percentage of opportunities open to all.svg',
+          mobileImage: 'Percentage of opportunities open to all.svg',
```
[the remark this answers is not in the document]
Comment (Revit) [reply apps/marketplace/pages/ReportPage.js]: changed as suggested

File: apps/marketplace/pages/ReportPage.js [hunk 3]
```diff
@@ -36,15 +36,15 @@ const itemData = {
...
+          imageAlt: '68% of all opportunities have been open to all'
         },
         {
           text: null,
-          image: 'Percentage of briefs for digital specialists.svg',
-          mobileImage: 'Percentage of briefs for digital specialists.svg',
-          imageAlt: '63% of briefs have been for digital specialists'
+          image: 'Percentage of opportunities for digital specialists.svg',
+          mobileImage: 'Percentage of opportunities for digital specialists.svg',
```
[the remark this answers is not in the document]
Comment (Revit) [reply apps/marketplace/pages/ReportPage.js]: changes as suggested

File: apps/marketplace/pages/ReportPage.js [hunk 4]
```diff
@@ -134,9 +135,9 @@ const itemData = {
         {
           text: null,
           fullWidth: true,
-          image: '768 Average responses per brief.svg',
-          mobileImage: '767 Average responses per brief.svg',
-          imageAlt: 'Buyers receive on average 12 responses per brief as of January 2018'
+          image: '768 Average responses per opportunity.svg',
+          mobileImage: '767 Average responses per opportunity.svg',
```
[the remark this answers is not in the document]
Comment (Revit) [reply apps/marketplace/pages/ReportPage.js]: changed as suggested

File: apps/marketplace/components/Reports/ReportItem.js [hunk 5]
```diff
@@ -18,10 +18,10 @@ const ReportItem = props => {
...
     const imgClass = includes(
       [
-        '768 Briefs by phase',
-        '767 Briefs by phase',
-        '768 Average responses per brief',
-        '767 Average responses per brief',
+        '768 Opportunities by phase',
+        '767 Opportunities by phase',
+        '768 Average responses per opportunity',
+        '767 Average responses per opportunity',
```
[the remark this answers is not in the document]
Comment (Revit) [reply apps/marketplace/components/Reports/ReportItem.js]: changes as suggested

File: src/bundles/CaseStudy/components/SubmitConfirmation/SubmitConfirmation.js [hunk 6]
```diff
@@ -30,7 +30,7 @@ class SubmitConfirmation extends React.Component {
                             <span styleName="callout-heading">You have been prioritised for assessment</span>
                         </h1>
                         <p>
-                            Before you can respond to this brief we need to assess your expertise in {domain}.
+                            Before you can respond to this opportunities we need to assess your expertise in {domain}.
```
Comment (tobiasbrown) [inline src/bundles/CaseStudy/components/SubmitConfirmation/SubmitConfirmation.js:33]:
```suggestion
                            Before you can respond to this opportunity we need to assess your expertise in {domain}.
```

File: apps/marketplace/components/Brief/BriefTrainingResponseSubmitted2.js [hunk 7]
```diff
@@ -7,7 +7,7 @@ import { rootPath } from 'marketplace/routes'
 
 const BriefTrainingResponseSubmitted2 = props => (
   <div className="row">
-    <DocumentTitle title="Brief Response Submitted - Digital Marketplace">
+    <DocumentTitle title="Opportunity Response Submitted - Digital Marketplace">
```
[the remark this answers is not in the document]
Comment (Revit) [reply apps/marketplace/components/Brief/BriefTrainingResponseSubmitted2.js:10]: changed as suggested in BriefSpecialistResponseSubmitted2.js

**Review by tobiasbrown — APPROVED** [review]

**Pull request merged**